keyboards/converter/rs_pcaz1_usb/matrix.c

```diff
@@ -0,0 +1,344 @@
+/*
```

Comment:
Could this be converted to matrix lite? 

https://docs.qmk.fm/#/custom_matrix?id=lite